### PR TITLE
Updates for brand colours, logo, and jobs hero

### LINKED ISSUE #4099: Update logo and brand colors to the new versions
Brand design has updated our logo and brand colors. We need to update them within the product and design system as well. 

| Color  | Current | New |
| ------------- | ------------- | ------------- |
| Blue: | `#00B4F2` |      `#00CBEC`
| Red/orange: | `#F96216` |      `#FF5543`
| Purple: | `#A305E1` |      `#A112FF`

See our documented brand guidelines [here](https://about.sourcegraph.com/handbook/marketing/brand/brand_guidelines).
Duplicate of the https://github.com/sourcegraph/sourcegraph/issues/22184 for this repo.

### LINKED ISSUE #5117: Center header image on /jobs
This image is left aligned on larger screen sizes:
image

This is towards the top of the page on [about.sourcegraph.com/jobs](https://about.sourcegraph.com/jobs) - can we center it?

### PR DESCRIPTION
**Pull request by bretthayes**
### What should this PR do?

- Closes #4099
- Closes #5117 
- Also fixes bug with improper container styles pushing viewport on the jobs page

### CODE REVIEW

File: website/src/css/pages/_jobs.scss [hunk 1]
```diff
@@ -35,7 +35,7 @@
         display: none;
 
         @media only screen and (max-width: 768px) {
-            display: block;
+            display: inline-block;
```
Comment (katjuell) [inline website/src/css/pages/_jobs.scss:38]: I hesitate to suggest this (considering how all over the place the margins and padding are), but would we want margin on the mobile image, to at least sort of match the text blocks and other items on the page?

**Review by katjuell — CHANGES_REQUESTED** [review]
It feels good to see a PR like this — thanks for getting those variables in place! 

Just one small question for you on this.

**Review by marybelzer — APPROVED** [review]

File: website/src/css/pages/_jobs.scss (hunk 1 above)
Comment (bretthayes) [reply website/src/css/pages/_jobs.scss:38]: Yeah, there's a lot of inconsistencies and style implementations that we need to clean up.

This should probably be a question for design but I do like the full-bleed images. We could remove the max-width constraint for the mobile image so from mobile up to the 768px breakpoint, the mobile image is still full-bleed. What do you think @stephzabala? Do you like the full-bleed images or do you want them floating within certain width constraints as they are now? Check [the jobs page](https://deploy-preview-5119--sourcegraph.netlify.app/jobs/) for responsiveness and [this loom](https://www.loom.com/share/88ff27b77a0d4e63be716c3765880c38).

Comment (bretthayes) [reply website/src/css/pages/_jobs.scss:38]: @stephzabala Great, sounds good! Yes, it's a lot more fluid from breakpoint to breakpoint and the images are full-bleed to the horizontal edges of the viewport 🤓

**Review by katjuell — APPROVED** [review]
lgtm!

**Pull request merged**